js/pages/Router.js

```diff
@@ -78,7 +78,7 @@ define(
         }, {});
         // anyway, we should track the moment when the user exits and check permissions once again
 				authApi.isAuthenticated.subscribe((isAuthenticated) => {
-					if (!isAuthenticated) {
+					if (!isAuthenticated && this.getModel().activePage() !== 'Feedback' && this.getModel().activePage() !== 'Home') {
```

Comment:
const NON_SECURE_PAGES = ['Home', 'Feedback'];
NON_SECURE_PAGES.includes(this.getModel().activePage())